domain/user/state/state.go

```diff
@@ -291,6 +291,20 @@ func (st *State) SetPasswordHash(ctx context.Context, uuid user.UUID, passwordHa
 	})
 }
 
+func CreateAdminUser(ctx context.Context, tx *sqlair.TX) error {
+	uuid, err := user.NewUUID()
+	if err != nil {
+		return errors.Annotate(err, "generating UUID")
+	}
+
+	adminUser := user.User{
+		Name:        "admin",
+		DisplayName: "admin",
+		CreatedAt:   time.Now(),
```

Comment:
@manadart we probably need to ensure that all times are UTC in the domain layer. That way we can move between system time and UTC easily.

Reply:
Do we need to? They're going to be stored with time zone information in state and when they get hydrated back into `time.Time` they will carry that information again. We are always going to be using `time.Time` comparison funcs so it should be safe and deterministic the whole way through?